core/lib/multistore.ts

```diff
@@ -234,9 +241,10 @@ export function localizedRoute (routeObj: LocalizedRoute | string | RouteConfig
     if (typeof routeObj !== 'object') {
       return '/' + storeCode + routeObj
     }
-    if (routeObj.name) {
-      routeObj.name = storeCode + '-' + routeObj.name
-    }
+    // TODO - need route name prefix?
+    // if (routeObj.name) {
+    //   routeObj.name = storeCode + '-' + routeObj.name
```

Comment:
I think it's still required for `setupMultistoreRoutes` which is adding the static routes - I belive we should optimize `setupMultistoreRoutes` the same way we did with `prepareDynamicRoutes` to add the routes just for the current storeView; shouldn't we?